src/main/CommandHandler.cpp

```diff
@@ -559,6 +564,11 @@ CommandHandler::tx(std::string const& params, std::string& retStr)
     ZoneScoped;
     std::ostringstream output;
 
+    if (mApp.getState() != Application::State::APP_SYNCED_STATE)
```

Comment:
This is a breaking change for Horizon ; do we really need this as the only thing it's doing is to stop accumulating transactions in memory

Reply:
No, we do not need this. I will revert it.